### PR TITLE
[1LP][RFR] Added test test_custom_button_ansible_credential_list

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: -v -k "test_custom_button_ansible_credential_list" --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7081f3b004095032c3f31cf31f8291af69d7c92d:

`cfme/tests/ansible/test_embedded_ansible_services.py`:
- :red_circle: Line 17:1: **F401** *'cfme.utils.version.current_version' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by quarckster** [comment]
58z failed due some timeout error not related to the changes in the PR.

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 1]
```diff
@@ -125,6 +128,27 @@ def service(appliance, ansible_catalog_item):
         service_.delete()
 
 
+@pytest.fixture
```
Comment (mshriver) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: I know its deprecated, but we use `yield_fixture` decorator everywhere else. I'd like to keep it consistent until we decide to drop it and just use fixture everywhere.

**Comment by dajoRH** [comment]
Lint report for commit 865d850b5f2c9944789e580439d7fe0a2c6203ae:

`cfme/ansible/repositories.py`:
- :red_circle: Line 6:1: **F401** *'widgetastic.utils.VersionPick' imported but unused*
- :red_circle: Line 17:1: **F401** *'cfme.utils.version.Version' imported but unused*

`cfme/services/requests.py`:
- :red_circle: Line 7:1: **F401** *'widgetastic.utils.VersionPick' imported but unused*
- :red_circle: Line 19:1: **F401** *'cfme.utils.version.Version' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5cec625714eda04b7910782ad596e6dbece91e9b:

`cfme/ansible/repositories.py`:
- :red_circle: Line 6:1: **F401** *'widgetastic.utils.VersionPick' imported but unused*
- :red_circle: Line 17:1: **F401** *'cfme.utils.version.Version' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**